pkg/controller/rsm/transformer_object_generation.go

```diff
@@ -139,19 +138,10 @@ func (t *ObjectGenerationTransformer) Transform(ctx graph.TransformContext, dag
 		}
 	}
 	handleDependencies := func() {
-		if rsm.Spec.RsmTransformPolicy == workloads.ToPod {
```

Comment:
if policy is rsm，dont need to add pod to dependency?

Reply:
rsm(root object)->pod dependency will be added by default